tests/test_gpu_common.py

```diff
@@ -66,6 +67,23 @@ def test_maxpar_option(self):
         assert trees[0][0] is trees[1][0]
         assert trees[0][1] is not trees[1][1]
 
+    @pytest.mark.parametrize('dtype', [np.complex64, np.complex128])
```

Comment:
What happens if you try to take derivatives of an expression containing the imaginary unit? Something like `(sympy.I*u).dx`?

Reply:
Sympy.I is just a sympy Atomic it's treated like any other symbol or number such as S.One or S.NegativeOne

Reply:
Sure, I'm just inclined to add tests to check that things work the way they 'should' since I've been tripped up in the past